src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
+$calendarFont: arial;
+$fontSize: 30px;
+$daySize: 100px;
+$dayGap: 1px;
+$totalSize: 7 * $daySize;
+$totalGap: 6 * $dayGap;
```

Comment:
let's move vars to a separate file and import it.

Reply:
Make sure you fixed all previous comments 
![image](https://github.com/mate-academy/layout_calendar/assets/99403198/67cea184-a57f-414e-a56c-8fc54122d094)